data.md

```diff
@@ -53,10 +53,10 @@ Some important numbers that are referred to often during execution:
 -   `chop` interperets an integers modulo $2^256$.
 
 ```{.k .uiuck .rvk}
-    syntax Int ::= chop ( Int ) [function]
- // --------------------------------------
-    rule chop ( I:Int ) => I %Int pow256 requires I <Int 0  orBool I >=Int pow256
-    rule chop ( I:Int ) => I             requires I >=Int 0 andBool I <Int pow256
+    syntax Int ::= chop ( Int ) [function, smtlib(chop)]
```

Comment:
add a comment saying what `smtlib(chop)` means?

Reply:
um, are you asking what's the `smtlib` attribute? it is an attribute to be used when you encode this term into z3 smt query, since z3 doesn't not allow all of the characters we allowed for a name of functions/constants.

Reply:
if you're asking to put a comment what `smtlib` means, i think we should do it somewhere else, because the attribute is used other places as well throughout the entire semantics.